extensions/amp-story/1.0/amp-story-render-service.js

```diff
@@ -15,8 +15,16 @@
  */
 
 import {CommonSignals} from '../../../src/common-signals';
+import {Services} from '../../../src/services';
+import {user} from '../../../src/log';
 import {whenUpgradedToCustomElement} from '../../../src/dom';
 
+/**
+ * Maximum milliseconds to wait for service to load before logging a warning.
+ * @const
+ */
+const LOAD_TIMEOUT = 3000;
```

Comment:
This needs to be shorter than the render delay timeout to account for the latency downloading and executing the amp-story js.